src/router/Routes.tsx

```diff
@@ -28,12 +30,10 @@ const Router = () => {
             <div className="App">
                 <Switch>
                     <ReverseAuthRoute path={"/login"} component={Login}/> //Requires not being logged in
-                    <AdminRoute path={"/book"} component={() => <ContainedComponent children={Book} isAdmin={true} selected={0}/>}/>
-                    <AuthRoute path={"/userHome"} component={Logged}/> //Requires being logged in
-                    <Route path={"/signup"} component={signUp}/>
-                    <AdminRoute path={"/adminHome"} component={AdminHome}/> //Requires admin role
-                    <Route path={'/profile'} component={ProfileView}/>
-                    <Route path={"/home"} component={Home}/>
+                    <AuthRoute path={"/book"} component={() => <ContainedComponent children={() => <Book isAdmin={isAdmin}/>} isAdmin={isAdmin} selected={0}/>}/>
+                    <Route path={"/signup"} component={SignUp}/>
+                    <AuthRoute path={'/profile'} component={ProfileView}/>
```

Comment:
When the route has been selected, the item on the sidebar should be colored. Check the `/book` route where this is accomplished.